pod/perldelta.pod

```diff
@@ -61,6 +61,13 @@ L<ExtUtils::MakeMaker> has been upgraded from version 7.42 to 7.44.
 
 =item *
 
+L<feature> has been upgraded from version 1.57 to 1.58.
+
+A new C<indirect> feature has been added, which is enabled by default
+but allows turning off indirect method syntax.
```

Comment:
This (and the feature entry) could link to https://perldoc.pl/perlobj#Indirect-Object-Syntax

Reply:
Possibly also that section should be updated to mention the feature